### PR TITLE
Call new disable remote method from model class [2.x]

### PR DESCRIPTION
**Pull request by richardpringle**
backport of https://github.com/strongloop/loopback/commit/0ab33a82d4df767d26077079701d9af15a08c84b

### CODE REVIEW

**Comment by superkhau** [comment]
@richardpringle Add note for you to remember to remove the deprecation message in the backport too.

File: test/model.test.js [hunk 1]
```diff
@@ -707,7 +707,21 @@ describe.onServer('Remote Methods', function() {
       var callbackSpy = require('sinon').spy();
       var TestModel = app.models.TestModelForDisablingRemoteMethod;
       TestModel.on('remoteMethodDisabled', callbackSpy);
-      TestModel.disableRemoteMethod('findOne');
+      TestModel.disableRemoteMethod('findOne', true);
```
Comment (bajtos) [inline test/model.test.js]: In 2.x, the method `disableRemoteMethod` is not deprecated. Can we place keep this test unmodified to cover this API?

**Comment by richardpringle** [comment]
@bajtos, `findOne` is a static method, the `true` should have always been there.

**Comment by bajtos** [comment]
> @bajtos, `findOne` is a static method, the `true` should have always been there.

Ah, I see, you are right.

**Comment by richardpringle** [comment]
@bajtos, shit should be okay to merge right? Those windows builds aren't failing anymore and it doesn't seam like the failed tests for the downstream buils have anything to do with this change.

**Comment by superkhau** [comment]
Skipping multitenancy tests to get CI green at https://github.com/strongloop/loopback-multitenancy/pull/11.

**Comment by bajtos** [comment]
@slnode test please

**Comment by bajtos** [comment]
> okay to merge right? Those windows builds aren't failing anymore and it doesn't seam like the failed tests for the downstream buils have anything to do with this change.

Agreed, go ahead and :shipit:

**Pull request merged**